From e81222780096f2e15280b2b7c47e96c36f98d281 Mon Sep 17 00:00:00 2001
From: freedesktop_sdk_updater <freedesktop_sdk_updater@libreml.com>
Date: Thu, 20 Aug 2026 02:29:48 +0000
Subject: [PATCH] Update elements/include/ostree-source.yml to v2026.4-0

Updates elements/include/ostree-source.yml to v2026.4-0 (4f4445a)
Previous version was v2026.3-0 (867d4e0)
---
 elements/include/ostree-source.yml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/elements/include/ostree-source.yml b/elements/include/ostree-source.yml
index 3a29a71290..680f97cac9 100644
--- a/elements/include/ostree-source.yml
+++ b/elements/include/ostree-source.yml
@@ -4,7 +4,7 @@ sources:
   track: v*
   exclude:
   - v*.*.*
-  ref: v2026.3-0-g867d4e000137d466e06c5173691458e0b4a38a26
+  ref: v2026.4-0-g4f4445a8912f70c604fa84a057a8f4e904e35a24
 - kind: git_module
   path: libglnx
   url: gnome:libglnx.git
-- 
GitLab

